packages/login/network/mnemonicProvider.js

```diff
@@ -10,14 +10,18 @@ import NemWallet from './NemWallet'
 class MnemonicProvider {
   getMnemonicProvider (mnemonic, { url, network } = {}) {
     const ethereumWallet = this.createEthereumWallet(mnemonic, network, url)
-    const btc = network && network.bitcoin && this.createBitcoinWallet(mnemonic, bitcoin.networks[ network.bitcoin ])
-    const bcc = btc
+    const btc = network && network.bitcoin && this.createBitcoinWallet(mnemonic, bitcoin.networks[network.bitcoin])
```

Comment:
throw if network not setted

Reply:
No =(
See for example settings for the bitcoingold_testnet (it is commnted out).
It is ok not to have a network.
In this case the relatet "engine" will be null
And the Provider will not be isInitialized

Won't fix.